content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -5,7 +5,7 @@ category: verify-your-identity
 permalink: /help/verify-your-identity/verify-your-identity-in-person/
 order: 7
 ---
```

Comment:
**General design feedback:** 
@svalexander For the h2 "Complete the process at a participating Post Office before your deadline", can you include a line break so that “your deadline” is on the second line at the largest width?

Reply:
On line 50 the translated copy in the code reads as "Complete el proceso en la oficina de correos antes la fecha límite" but in the preview it's "Complete el proceso en una oficina de correos adherida antes de la fecha límite"

Reply:
only for the en version? or also es + fr? if the latter should the break be before  "la fecha límite" and "la date limite."?

Reply:
Looks good otherwise! Thank yooouuuu :)

Reply:
`Complete el proceso en la oficina de correos antes la fecha límite` is the text that was removed.

Reply:
I only noticed the strange line break in the English version

Reply:
just pushed up the line break change